lib/Models/Catalog/CatalogReferences/MagdaReference.ts

```diff
@@ -179,7 +187,8 @@ export default class MagdaReference extends AccessControlMixin(
         this.uniqueId,
         existingRecord,
         override,
-        previousTarget
+        previousTarget,
+        addOrOverrideAspects
```

Comment:
At some point I think we should refactor all these arguments, there are too many of them and they are quite hard to follow

Reply:
But I think that can be left for another day